web/packages/core/src/load-options.ts

```diff
@@ -117,6 +117,39 @@ export const enum WindowMode {
     Gpu = "gpu",
 }
 
+/**
+ * The render backend of a Ruffle player.
+ */
+export const enum RenderBackend {
+    /*
+     * A draft API that is currently unavailable, but will be preferred if available in the future.
+     * Should behave the same as wgpu-webgl, except with lower overhead and thus better performance.
+     */
+    WebGpu = "webgpu",
+
+    /*
+     * The most featureful and currently preferred backend.
+     * Rendering is done the same way as in the desktop app, then translated to WebGL on-the-fly.
+     */
+    WgpuWebgl = "wgpu-webgl",
+
+    /*
+     * A vanilla WebGL backend. Was previously the default backend,
```

Comment:
I'd change this to "Was the preferred backend until the switch to wgpu-webgl at the end of 2022"